### PR TITLE
Adding changes to include a test for passing a specific `K8S_VERSION` to the cluster.

### PR DESCRIPTION
**Pull request by sowmyav27**
Adding changes to include a test for passing a specific `K8S_VERSION` to the cluster.

### CODE REVIEW

File: tests/validation/tests/rke/test_install_config.py [hunk 1]
```diff
@@ -6,7 +6,7 @@ def test_install_config_1(test_name, cloud_provider, rke_client, kubectl):
     """
     Node Address specified as just IP and using only this in the node spec
```
Comment (soumyalj) [inline tests/validation/tests/rke/test_install_config.py:7]: Could you add a comment that a specific K8s version is being tested in this test case ? We are replacing cluster_install_config_1.yml.j2 with cluster_install_config_11.yml.j2 . Do we still want to keep cluster_install_config_1.yml.j2 ?

File: tests/validation/lib/rke_client.py [hunk 2]
```diff
@@ -54,7 +55,9 @@ def build_rke_template(self, template, nodes, **kwargs):
         """
         render_dict = {
             'master_ssh_key_path': self.master_ssh_key_path,
-            'network_plugin': DEFAULT_NETWORK_PLUGIN}
+            'network_plugin': DEFAULT_NETWORK_PLUGIN,
+            'k8s_version': K8S_VERSION
+        }
```
Comment (soumyalj) [inline tests/validation/lib/rke_client.py:60]: Could we also run the other test_install_config tests to see if they are working ? Its a change in the method build_rke_template()  of rke_client.py. Does this change affect other tests ?

File: tests/validation/tests/rke/test_install_config.py (hunk 1 above)
Comment (sowmyav27) [reply tests/validation/tests/rke/test_install_config.py:7]: Done.
Yes `cluster_install_config_1.yml.j2` is still available as its being used by another test.

File: tests/validation/lib/rke_client.py (hunk 2 above)
Comment (sowmyav27) [reply tests/validation/lib/rke_client.py:60]: Does not affect other tests.

**Comment by sangeethah** [comment]
LGTM

**Pull request merged**